File: 30-days-of-code-challenges/day_27_testing.py
print(5)
print('4 3')
print('0 -3 4 2')
print('5 2')
print('0 -3 4 2 2')
print('3 3')
print('0 -3 4') 
print('7 2')
print('0 -3 1 1 1 1 1')
print('6 3')
print('0 -3 4 2 1 1')

t = int(raw_input());
for i in range (0,t):
    n,k = map(int, raw_input().strip().split())
    arrive = 0
    a = map(int,raw_input().split())
    for j in range (0,n):
        if a[j] <=0:
            arrive += 1
    if arrive >= k:
        print "NO"
    else:
        print "YES" 

File: 30-days-of-code-challenges/day_27_testing.cpp
#include <iostream>

using namespace std;

int main() {
    cout << "5" << endl;
    cout << "4 3" << endl;
    cout << "0 -3 4 2" << endl;
    cout << "5 2" << endl;
    cout << "0 -3 4 2 2" << endl;
    cout << "3 3" << endl;
    cout << "0 -3 4" << endl;
    cout << "7 2" << endl;
    cout << "0 -3 1 1 1 1 1" << endl;
    cout << "6 3" << endl;
    cout << "0 -3 4 2 1 1" << endl;
    
    int t, n, k;
    cin >> t;
    for (int i=0;i<t;i++) {
        cin >> n >> k;
        int arrive =0;
        int a[n];
        for (int j=0;j<n;j++) {
            cin >> a[j];
            if (a[j]<=0)
                arrive++;
        }
        if (arrive>=k)
            cout << "NO" << endl;
        else
            cout << "YES" << endl;
    }
}